public/app/features/alerting/unified/components/export/GrafanaReceiversExporter.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+import { LoadingPlaceholder } from '@grafana/ui';
+
+import { alertRuleApi } from '../../api/alertRuleApi';
+
+import { FileExportPreview } from './FileExportPreview';
+import { GrafanaExportDrawer } from './GrafanaExportDrawer';
+import { ExportFormats, jsonAndYamlGrafanaExportProviders } from './providers';
+
+interface GrafanaReceiversExportPreviewProps {
+  exportFormat: ExportFormats;
+  onClose: () => void;
+  decrypt: string;
```

Comment:
Shouldn't `decrypt` be of type `boolean`?

Reply:
endpoint requires a string

Reply:
yes, not sure why i put an string? 🤔